admin/test/features/step_definitions/generate_pupil_pins_steps.rb

```diff
@@ -0,0 +1,15 @@
+
+And(/^I am on the generate pupil pins page$/) do
+  school_landing_page.generate_pupil_pin.click
+  @page = generate_pupil_pins_page
```

Comment:
what is the value of saving into a instance variable here? @sinhaamit1980

Reply:
Done